package/gluon-mesh-babel/check_site.lua

```diff
@@ -1,4 +1,3 @@
-need_string_match(in_domain({'node_prefix6'}), '^[%x:]+/64$')
```

Comment:
In my opinion we should not duplicate checks that are already provided by dependencies. It may even make sense to have the l3-common package check values it doesn't use itself if all actual L3 protocol package need them, so we consolidate common checks.

Reply:
So what do now?

Reply:
I've moved node_prefix6. Is that enough? Anything else left?